barretenberg/cpp/src/barretenberg/commitment_schemes/shplonk/shplemini.hpp

```diff
@@ -464,7 +456,7 @@ template <typename Curve> class ShpleminiVerifier_ {
                                                          std::vector<Fr>& scalars,
                                                          Fr& constant_term_accumulator)
     {
-        const size_t virtual_log_n = gemini_neg_evaluations.size();
+        const size_t virtual_log_n = use_padding ? gemini_neg_evaluations.size() : log_n;
```

Comment:
By the use in this function it doesn't look like use_padding needs to be a constexpr. it could be a defaulted param of this function.